### ISSUE TITLE
@n8n/chat: loadPreviousSession returns 500

### ISSUE THREAD

**Issue by Morriz** [n8n-io/n8n#11609]
### Bug Description

When using `@n8n/chat` and requesting `action: loadPreviousSession` when the memory store is empty, we receive an http 500 error, which is not sane. The request is valid and the api should just return an empty list: `[]`.

### To Reproduce

1. implement `@n8n/chat`

### Expected behavior

the html status code to be 200 and to get an empty list

### Operating System

Docker

### n8n Version

1.65.2

### Node.js Version

Docker

### Database

SQLite (default)

### Execution mode

main (default)

**Comment by Joffcom** [comment]
This is now in Linear and is being tracked as `N8N-7898`

**Comment by jeanpaul** [comment]
Hey @Morriz, are you connecting the memory store to your chat-trigger? Because if you have an (empty) memory connected to your chat-trigger, it should just return an empty list. Otherwise, it's correctly returning an error, because your workflow is incorrectly configured (the memory-node should be connected to your chat trigger if you've enabled `loadPreviousSession`).

**Comment by Morriz** [comment]
You are correct. I am not isolating my tests enough, because then I'd know ;p
